### PR TITLE
Update Dockerfile

### LINKED ISSUE #33: Too long startup time?
```
when I start your docker images,It need more than 5mins to access the page;
I have a good suggestion,
change this file:
/usr/local/openjdk-11/conf/security/java.security
securerandom.source=file:/dev/random   to  securerandom.source=file:/dev/urandom,
It starts right away
```
Looking forward to further communication;

### PR DESCRIPTION
**Pull request by zengxc-1996**
I try to add this line to the dockerfile, which is very useful. The page will no longer wait, which is very important for us to use your image. I hope you can seriously consider this scheme,Or do you have better suggestions . I look forward to your reply.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,6 +17,6 @@ VOLUME /cmak/conf
 ENV JAVA_OPTS=-XX:MaxRAMPercentage=80
 
 WORKDIR /cmak
-
+RUN sed -i 's/\/dev\/random/\/dev\/.\/urandom/g' /usr/local/openjdk-11/conf/security/java.security
 ENTRYPOINT ["/cmak/bin/cmak", "-Dpidfile.path=/dev/null", "-Dapplication.home=/cmak", ""]
```
Comment (hleb-albau) [inline Dockerfile:21]:
```suggestion
ENTRYPOINT ["/cmak/bin/cmak", "-Dpidfile.path=/dev/null", "-Dapplication.home=/cmak","-Dsecurerandom.source=file:/dev/urandom", ""]
```

**Comment by hleb-albau** [comment]
Could, you, please test one more scenario - add -D option in entrypoint. See my comment-suggestion 

That solution is better. If we change the base java image, paths for java installation can be changed too.

**Comment by zengxc-1996** [comment]
Yes, I've tested it. I'm sorry, I made a mistake. It should be changed to 
`-Dsecurerandom.source=file:/dev/./urandom`
I will do this test again after you modify it
thanks!

**Comment by hleb-albau** [comment]
> Yes, I've tested it. I'm sorry, I made a mistake. It should be changed to
> `-Dsecurerandom.source=file:/dev/./urandom`
> I will do this test again after you modify it
> thanks!

`-Dsecurerandom.source=file:/dev/./urandom` is equalent to `-Dsecurerandom.source=file:/dev/urandom` in unix systems. So there is no mistake.

**Comment by eshepelyuk** [comment]
I would suggest to put this JVM argument into `JAVA_OPTS`, not to ENTRYPOINT.

**Comment by hleb-albau** [comment]
Closed, see the linked issue for details

**Pull request closed without merging**